src/Features/Core/Portable/TodoComments/TodoCommentDescriptor.cs

```diff
@@ -24,28 +26,18 @@ public TodoCommentDescriptor(string text, int priority)
             Priority = priority;
         }
 
-        public static ImmutableArray<TodoCommentDescriptor> Parse(string data)
+        public static ImmutableArray<TodoCommentDescriptor> Parse(ImmutableArray<string> items)
```

Comment:
https://github.com/xamarin/vsmac/pull/8604 would indicated that TypeScript has IVT usage of this. Will try to confirm